### PR TITLE
qemu.tests: Support Windows guests in virtual_nic_private test

### PR DESCRIPTION
**Pull request by ypu**

### CODE REVIEW

**Comment by ypu** [comment]
ID: 1132850

File: qemu/tests/cfg/virtual_nic_private.cfg [hunk 1]
```diff
@@ -1,7 +1,7 @@
 - virtual_nic_private:
     virt_test_type = qemu
     #now this case only support linux guest, will support windows later
-    only Linux
+    #only Linux
```
Comment (suqinhuang) [inline qemu/tests/cfg/virtual_nic_private.cfg]: How about delete line3 and line4 directly.

**Comment by ypu** [comment]
Thanks @suqinhuang, removed the lines. Can you help to check it again?

**Comment by spcui** [comment]
LGTM
Acked-by: @spcui

**Comment by ypu** [comment]
Thanks @spcui will merge it.

**Pull request merged**